src/main/java/it/pagopa/pm/gateway/service/CcResumeStep2Service.java

```diff
@@ -66,9 +61,7 @@ public class CcResumeStep2Service {
     private ObjectMapper objectMapper;
 
     @Autowired
-    public CcResumeStep2Service(ClientsConfig clientsConfig) {
-        this.clientsConfig = clientsConfig;
-    }
+    private VposPatchUtils vposPatchUtils;
```

Comment:
Prefer constructor injection to field injection

Reply:
Done https://github.com/pagopa/pagopa-payment-transactions-gateway/pull/239/commits/d7187758c5845f94735a249624d783b9e9531754